src/convertToObject.js

```diff
@@ -5,8 +5,25 @@
  *
  * @return {object}
  */
+
 function convertToObject(sourceString) {
-  // write your code here
+  const cleanSource = sourceString.trim();
+
+  const declarations = cleanSource.split(';');
+
+  const styleObject = {};
+
+  declarations.forEach((declaration) => {
+    if (declaration.trim()) {
+      const [property, value] = declaration.split(':');
+
+      if (property && value) {
+        styleObject[property.trim()] = value.trim();
+      }
+    }
+  });
```

Comment:
[CODE PRACTICE] - Avoid using loops in this task, use iteration methods instead. Here, `forEach` is acceptable, but you could consider using `reduce` to build the `styleObject` directly, which would be more in line with functional programming practices.